### PR TITLE
Pull-Request: hello-rails-back-end

### PR DESCRIPTION
**Pull request by Zven94**
### **hello-rails-back-end**

Implement a connection between a Ruby on Rails back-end and React front-end:

- [x] Create a new Rails API app called 'hello-rails-back-end'.
- [x] Initialize your project with Git.
- [x] Make sure that your project has a Postgres database set up.
- [x] Create a table for storing your messages and create 5 different greetings.
- [x] Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).

https://github.com/Zven94/hello-rails-back-end/assets/125831572/68c5371b-3d70-494e-96e7-90a5c8391c47

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  "Welcome to our app!",
+  "It is a pleasure having you in here.",
+  "Thank you for joining us.",
+  "Welcome to the community.",
+  "We hope you feel at home."
+]
+
+messages.each do |content|
+  Message.create(content: content)
+end
```
Comment (FrederickMih) [inline db/seeds.rb:11]: - Nice job for seeding your table with greeting messages but unfortunately the migration file for creating the table doesn't exit at all

**Review by FrederickMih — COMMENTED** [review]
## Hi :raising_hand: :raising_hand: @Zven94, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Linters checks are not running on the frontend PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/seeds.rb (hunk 1 above)
Comment (MahmoudBakr23) [inline db/seeds.rb:11]: - [ ] To ensure a better and more valid review from your side, please make sure to create and run the migration file that's supposed to have been created with the Message model, this is also to make sure there's a schema (database) in which the messages will be created and fetched from. 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Zven94 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done on working with the two repo's configs and setup so far
- Well done with the controller in the RAILS API
- Well done setting up the REACT component with redux store and reducer setup
- Great job with the routes in both repos
- Descriptive PRs
- Excellent and extra effort in communication
- Correct Linters

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️

- [ ] See the inline comment in the front-end PR

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Zven94, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- "Hello world" project with `react-rails` gem ✔️
- Implement the `API` in the backend correctly ✔️
- Add link to the `Frontend` in the README file ✔️ 
- Good use of Git flow ✔️

## Optional changes

- [ ] I would kindly suggest you always make your commit messages more professional (i.e start with an uppercase)

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**